open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.extensions/src/main/java/com/wso2/openbanking/accelerator/consent/extensions/validate/util/ConsentValidatorUtil.java

```diff
@@ -69,6 +71,25 @@ public static JSONObject getValidationResult(String errorCode, String errorMessa
 
         return validationResult;
     }
+
+    public static JSONObject getErrorAndLog(JSONObject errorResult, ConsentValidationResult consentValidationResult) {
+
+        JSONObject validationResult = new JSONObject();
```

Comment:
why we create new validationResult? and this will always create true as validation result right?

Reply:
Fixed in [dc8a94b]

Reply:
Any reason creation of new validationResult?, along with consentValidationResult?